vite.config.ts

```diff
@@ -52,6 +52,7 @@ export default defineConfig({
           ["react-dom"]: "ReactDOM",
           ["@remix-run/react"]: "@remix-run/react",
         },
+        assetFileNames: "stylesheet[extname]",
```

Comment:
are you telling me I spent 2 days trying to figure out how to output the css file and this is all it took? 😮‍💨